build-wheels-new.sh

```diff
@@ -69,16 +66,18 @@ declare -a PYTHON_VERS=( $1 )
 for PYVER in ${PYTHON_VERS[@]}; do
     PYBIN="/opt/python/$PYVER/bin"
     PYVER_NUM=$($PYBIN/python -c "import sys;print(sys.version.split(\" \")[0])")
-    "${PYBIN}/pip" install -r /io/requirements.txt
+
+    ${PYBIN}/pip install -r /io/requirements.txt
     PYTHONVER="$(basename $(dirname $PYBIN))"
     BUILDDIR="/io/gtsam_$PYTHONVER/gtsam_build"
     mkdir -p $BUILDDIR
     cd $BUILDDIR
     export PATH=$PYBIN:$ORIGPATH
-    "${PYBIN}/pip" install cmake
+    ${PYBIN}/pip install cmake
 
     PYTHON_EXECUTABLE=${PYBIN}/python
-    PYTHON_INCLUDE_DIR=$( find -L ${PYBIN}/../include/ -name Python.h -exec dirname {} \; )
+    PYTHON_INCLUDE_DIR=$(${PYTHON_EXECUTABLE} -c "from distutils.sysconfig import get_python_inc; print(get_python_inc())")
```

Comment:
Hi @varunagrawal , thanks for the PR. Would you mind adding a bit of documentation inline explaining why distutils is the best (or only) way to get the path to the python3 binary and python lib dir?

And maybe also why we need to explicitly provide these two items to cmake?